helm/flowforge/Chart.lock

```diff
@@ -2,5 +2,5 @@ dependencies:
 - name: postgresql
   repository: https://charts.bitnami.com/bitnami
   version: 11.9.13
-digest: sha256:ea11617d363e174146b33ceafb5e2072d36731c2446178a91e5e6f705d0db067
-generated: "2023-11-15T14:44:27.496269+01:00"
+digest: sha256:1423e37aa789fccab1a4b5db593ccda3887032de2a4814b928df938f6db1de3a
+generated: "2024-02-13T12:15:48.820039+01:00"
```

Comment:
Why has this changed as part of this PR? should it have changed earlier when we updated the PostgreSQL chart version?

Reply:
Chart.lock had to be updated because content of `Chart.yaml` changed. Otherwise [linting fails](https://github.com/FlowFuse/helm/actions/runs/7885412540/job/21516491779?pr=307#step:7:23)

Reply:
And it was change when we updated PostgreSQL sub-chart version: https://github.com/FlowFuse/helm/commit/e07ff89ddcf0220fffc711b75a867884db16261b